snapcraft.yaml

```diff
@@ -4,7 +4,7 @@ summary: Gaia Daemon # 79 char long summary
 description: |
   This snap provides the Gaia daemon gaiad and the command line
   tool gaiacli.
-grade: devel # must be 'stable' to release into candidate/stable channels
+grade: stable
```

Comment:
Does this imply that gaia is out of beta?

Reply:
It just allows us to promote revisions to candidate/stable when we think it's ready

Reply:
This applies to snap, not to Gaia, right?

Reply:
This applies to gaia snap packages